hack/common.sh

```diff
@@ -70,22 +65,18 @@ build_WMCO() {
 # 1: path to the operator-sdk binary to use
 run_WMCO() {
   local OSDK=$1
-
-  # Create a temporary directory to hold the edited manifests which will be removed on exit
-  MANIFEST_LOC=`mktemp -d`
-  trap "rm -r $MANIFEST_LOC" EXIT
-  cp -r deploy/olm-catalog/windows-machine-config-operator/ $MANIFEST_LOC
-  sed -i "s|REPLACE_IMAGE|$OPERATOR_IMAGE|g" $MANIFEST_LOC/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml
+  # add operator image to csv
+  replace_image $OPERATOR_IMAGE
 
   # Validate the operator bundle manifests
-  $OSDK bundle validate "$MANIFEST_LOC"/windows-machine-config-operator/
+  $OSDK bundle validate deploy/olm-catalog/windows-machine-config-operator/
```

Comment:
Please add  `MANIFEST_LOC=deploy/olm-catalog/windows-machine-config-operator/` and use that everywhere.